bears/js/JSStandardBear.py

```diff
@@ -16,13 +18,52 @@ class JSStandardBear:
     """
 
     LANGUAGES = {'JavaScript', 'JSX'}
-    REQUIREMENTS = {NpmRequirement('standard', '10')}
+    REQUIREMENTS = {NpmRequirement('standard', '10'),
+                    NpmRequirement('typescript-eslint-parser', '12.0.0'),
+                    NpmRequirement('eslint-plugin-typescript', '0.8.1'),
+                    NpmRequirement('eslint-plugin-markdown', '1.0.0-beta.6'),
+                    NpmRequirement('eslint-plugin-html', '3.2.2')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax'}
     CAN_FIX = {'Formatting'}
     SEE_MORE = 'https://standardjs.com/rules.html'
 
-    def create_arguments(self, filename, file, config_file):
-        return (filename, '--verbose')
+    def create_arguments(self, filename, file, config_file,
+                         language: language = language('JavaScript'),
+                         global_vars: typed_list(str) = (),
+                         env: str = '',
+                         ):
+        """
+        :param global_vars:
+            The list of global variables in the code
+        :param language:
+            Set to ``Javascript`` by default. To check javascript in either
+            markdown or html specify ``Markdown`` or ``html``. Use
+            ``Typescript`` to lint typescript.
+        :param env:
+            To support global variables of certain environments. The complete
+            list of supported environments can be found here
+            https://eslint.org/docs/user-guide/configuring.HTML#specifying-environments
```

Comment:
404 with the `.HTML`

doc now at

https://eslint.org/docs/user-guide/configuring#specifying-environments

Note this settings should accept a typed_list(str) as it has many values.

might be useful to give this setting a more meaningful name, like `javascript_global_variable_environments` .  
We have a few ESLint based linters, and presumably this setting should be common to them all.
JSHint and JSLint also have a very similar concept.

Alternatively , we can name it the same as the inline comment syntax :  eslint-env or eslint_env